package.json

```diff
@@ -4,16 +4,20 @@
   "description": "Chrome extension adding extra features to the Bandcamp experience.",
   "main": "index.js",
   "dependencies": {
+    "webpack": "^4.43.0"
```

Comment:
Very minor note: I think Webpack could be considered a dev dependency, the [install docs](https://webpack.js.org/guides/installation/) note it as such. There is no real impact in this project, but as a general rule the packages listed in the `dependencies: {}` are critical to the runtime of the application, while dev- and build-time packages get listed in `devDependencies`.

If you decide to adjust this (again, not high priority) you could cut-paste the webpack line into `devDependencies`.